src/flags.rs

```diff
@@ -58,7 +78,20 @@ pub fn print_usage() {
 
 Environment variables:
 DENO_DIR           Set deno's base directory."
-  );
+    );
+  }
+}
+
+fn permission_prompt(message: String) -> DenoResult<()> {
+  if !atty::is(atty::Stream::Stdin) { return Err(permission_denied()) };
+
+  eprint!("{} Grant? yN", message);
```

Comment:
Is stderr the right thing to use here? I think you should use stdout.

Modify the line to have a space and question mark.
```
eprint!("{} Grant? y N ? ", message);
```

Reply:
The reasoning for stderr is if you're doing something like:

    $ deno file.ts > out.log

Then this will still work. What's the argument against?

Reply:
Ok, makes sense. Use stderr.

Reply:
Maybe add a comment about why you're using stderr - since that wasn't obvious to me.

Reply:
Sure, I guess you need stderr and stdin to be tty for this to work.